framework/SupervisedLearning/ARMA.py

```diff
@@ -1827,10 +1830,16 @@ def getGlobalRomSegmentSettings(self, trainingDict, divisions):
           targetVals = trainingDict[target][0]
           periods = np.asarray(self.fourierParams[target])
           full = periods[periods > (delta*self.nyquistScalar)]
-          segment[target] = periods[np.logical_not(periods > (delta*self.nyquistScalar))]
+          
+          #NOTE: train all the Fourier periods in the segment.
+          # segment[target] = periods
+
+          # # NOTE: orignially train the Fouirer whose periods shorter than segmentation length after segmentation. segment[target] store the shorter than segmentation length Fourier period.
+          segment[target] = periods[np.logical_not(periods > delta)]
```

Comment:
I'm concerned about this reversion of the feature Dylan added to give flexibility on the divider between "long" and "short" Fourier frequencies

Reply:
it will be readded. PS. I think that  "nyquistScalar" is not in the manual.